FoodOrder.TestApi/Controller/RestaurantTestController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using AutoMapper;
+using FluentAssertions;
+using FoodOrderApi.Controllers;
+using FoodOrderApi.DataProvider;
+using FoodOrderApi.Mappings;
+using FoodOrderApi.Model.DTO;
+using FoodOrderApi.TestApi.Mockdata;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using System.Net;
+
+namespace FoodOrderApi.TestApi.Controller
+{
+    public class TestRestaurantController
```

Comment:
Test classes are generally named in this format `<NameOfTheModuleYouTest>Tests`, for instance, if I am testing my Logger Module, the test will be named as `LoggerTests`

Reply:
Checked and updated the test classes names.